lib/Benchmark/Executor/MicrotimeExecutor.php

```diff
@@ -51,7 +57,8 @@ public function execute(Iteration $iteration, array $options = array())
             'revolutions' => $iteration->getRevolutions(),
             'beforeMethods' => var_export($subject->getBeforeMethods(), true),
             'afterMethods' => var_export($subject->getAfterMethods(), true),
-            'parameters' => var_export($iteration->getParameters(), true),
+            'parameters' => var_export($parameters, true),
+            'argString' => $this->getArgString(count($parameters))
```

Comment:
We construct the argument string for the template here to reduce overhead from calling `ReflectionMethod::invokeArgs`